src/lib/convert.ts

```diff
@@ -19,21 +20,34 @@ export interface ConvertionOptions {
   prefix: string;
   fileName: string;
   interfaceName: string;
-  srcDirectory: string;
+  srcDirectory: string[];
```

Comment:
Maybe we could rename it to srcDirecotries, what do you think?

Reply:
If also thought so.

Maybe I can rename here, but I think we should let the CLI parameter's name as it is.

Reply:
Yes - I also think so - let's rename it internally but keep the CLI as it is. So it makes sense if you want to pass in a single file and we are not introducing a brekaing change

Reply:
Done =)